### DISCUSSION TITLE
Get center or bounds of the newly created shape?

### DISCUSSION THREAD

**Discussion by bpolaszek** [geoman-io/leaflet-geoman#1179] [Questions & Help] [answered]
[Newbie inside]

Hello there! 👋 

I'm migrating from [leaflet-draw](https://github.com/Leaflet/Leaflet.draw) which is pretty buggy with Leaflet 1.8.x, but I couldn't figure out, whenever a shape is drawn (I assume, listening to the `pm:drawend` event), how to get its lat/lng bounds (in case of a polygon) or its center point + radius (in case of a circle).

Can someone guide me?

Thanks!
Ben

**Comment by Falke-Design** [comment, marked as answer]
You need to listen on the `pm:create` event on the map.

```
map.on('pm:create',(e)=>{
    console.log(e);
    if(e.layer instanceof L.Circle){
      console.log(e.layer.getRadius());
    }
});
````

**Reply by bpolaszek** [reply]
Thanks @Falke-Design ! 🙏
